### ISSUE TITLE
Failed to run 1_extract_hirid.py

### ISSUE THREAD

**Issue by mostafaalishahi** [USM-CHU-FGuyon/BlendedICU#6]
Many thanks for the fantastic job you have done.
I have failed to run the code for HiRID extraction, so I would appreciate it if you could provide any insights that can help me move forward.
I believe this specific line is leading to an error
```
timeseries = pd.read_csv(self.imputedstage_path,
                                 usecols=['imputed_stage/', 'reldatetime'])
```
As far as we have no imputed_stage/ as a column in our `imputedstage_path='imputed_stage/imputed_stage_csv.tar.gz'` but we have  imputed_stage/ as a folder in imputedstage_path.

here is the error

`ValueError: Multiple files found in TAR archive. Only one file per TAR archive: ['imputed_stage', 'imputed_stage/csv', 'imputed_stage/csv/part-82.csv', 'imputed_stage/csv/part-96.csv', .....`

**Comment by USM-CHU-FGuyon** [comment]
Thank you for your interest in our work ! 
This is due to the latest commit. This line did run in Pandas 1.4, but behaviour of read_csv changed in Pandas >1.5. I will fix this soon and keep you updated.

**Comment by USM-CHU-FGuyon** [comment]
The bug should be fixed in release v0.1.1 that was just uploaded. Please tell me if you find new difficulties running the codes.
